caas/kubernetes/provider/application/application_test.go

```diff
@@ -433,7 +433,7 @@ func getPodSpec() corev1.PodSpec {
 		ServiceAccountName:            "gitlab",
 		AutomountServiceAccountToken:  pointer.BoolPtr(true),
 		ImagePullSecrets:              []corev1.LocalObjectReference{{Name: "gitlab-nginx-secret"}},
-		TerminationGracePeriodSeconds: pointer.Int64Ptr(300),
+		TerminationGracePeriodSeconds: pointer.Int64Ptr(30),
```

Comment:
This is a pretty big change?

Reply:
This was because we were trying to paper over the sidecar teardown issue. This at 30s, the default, is uncontroversial.